### PR TITLE
Fixed documentation

### PR DESCRIPTION
**Pull request by AshitaSingamsetty**
Added extra line before "Available Steps" and corrected the ,'s and grammar of "allow_private" decription

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,8 @@ A bit more about the different fields...
 - `allow_private` (optional)
     - bool [true, false]
     - default: false
-    - If false bugs that are not public will not be synchronized. Note that in order to synchronize
-      private bugs the bugzilla user that JBI runs as must be in the security groups that are making
-      the bug private.
+    - False bugs will not be synchronized if they are not made public. Note that in order to synchronize private bugs,
```
Comment (leplatrem) [inline README.md:36]:
```suggestion
    - If `false`, bugs will not be synchronized if they are not made public. Note that in order to synchronize private bugs,
```

**Review by leplatrem — CHANGES_REQUESTED** [review]
Thank you!

File: README.md [hunk 2]
```diff
@@ -34,13 +34,12 @@ A bit more about the different fields...
...
-- `bugzilla_user_id`
-    - a bugzilla user id, a list of user ids, or a literal "tbd" to signify that no bugzilla user id is available
-    - If an issue arises with the workflow, communication will be established with these users
-    - Please enter the user information for one or more stakeholders
+    - If `false`, bugs will not be synchronized if they are not made public. Note that in order to synchronize private bugs,
+     the bugzilla user that JBI runs as must be in the security groups that are making the bug private.
+- `contact`
+    - an email address, a list of email addresses, or a literal "tbd" to signify that no contact is available
+    - If an issue arises with the workflow, communication will be established with these contacts
+    - Please enter the contact information for one or more stakeholders
```
Comment (leplatrem) [inline README.md:42]: I think there was an issue with the merge of changes made in #440 

I'm sorry but you would have to adjust the changes one more time :(

**Review by leplatrem — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AshitaSingamsetty) [reply README.md]: I have made modifications according to the new commits. Hope its fine now. Let me know if you want more changes.

**Review by leplatrem — APPROVED** [review]

**Comment by leplatrem** [comment]
Thank you @AshitaSingamsetty !

**Comment by bsieber-mozilla** [comment]
Thanks for this PR @AshitaSingamsetty; I've pulled the commits onto [this PR](https://github.com/mozilla/jira-bugzilla-integration/pull/490)

@leplatrem could you give an approval on the linked PR (it's set to auto-merge when approved).

I'm going to close this one for now; but feel free to reopen if that's not sufficient.

**Pull request closed without merging**